src/main/java/com/aws/greengrass/shadowmanager/ipc/UpdateThingShadowRequestHandler.java

```diff
@@ -242,9 +242,7 @@ public UpdateThingShadowHandlerResponse handleRequest(UpdateThingShadowRequest r
                             .withVersion(updatedDocument.getVersion())
                             .withClientToken(clientToken)
                             .withTimestamp(Instant.now())
-                            // explicitly convert to shadow document to return valid state.
-                            // this is to prevent edge cases like returning null
```

Comment:
this seems to think that returning null is bad... so do we think this is a problem for our customers? Does the cloud shadow service ever return null?

Reply:
my initial understanding was wrong, cloud shadow service does return null for update accepted topic.  verified using the IoT mqtt test client and a real shadow